easy/test_is_up.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python2
```

Comment:
out of curiosity: why `python2`?

Reply:
Because this script is run inside old distributions which ship with python 2